### ISSUE TITLE
[Proposal] Add trait for simplified testing

### ISSUE THREAD

**Issue by jwalton512** [tightenco/mailthief#22]
Add a trait, similar to the existing traits in use by [laravel's test case today](https://github.com/laravel/framework/tree/5.2/src/Illuminate/Foundation/Testing/Concerns).

An example below.  This is currently just a proof of concept, but I don't imagine it to be much more complex than it is now.

https://github.com/jwalton512/mailthief/commit/5d8cbd9855b51033d2ffcb55bf5f62f1c0b9c9d2

@mattstauffer @adamwathan thoughts?

**Comment by mattstauffer** [comment]
It reads pretty nicely 👌

**Comment by besologic** [comment]
@jwalton512, looks like a good start! If you do a PR we'd love to merge in your work.

**Comment by ivanvermeyen** [comment]
Cool, this is kinda what I also had in mind (see #18). Only I used method names like `seeEmailFor($email)` because it felt more like Laravel's `seeInDatabase` and `seePageIs` etc. Dunno if that would be better or not, probably just a personal preference :)

I noticed you didn't add `@before` to the `hijack` method. Would that cause issues in some cases? I added it in my own trait so I didn't have to call `hijack` manually everytime :)

Thanks for doing this work! 👍
